roles/group/shcmember/tasks/copy_ssl.yml

```diff
@@ -2,17 +2,18 @@
 
 - name: enable ssl module
   command: "a2enmod ssl"
```

Comment:
This should probably use the modprobe module instead of a command, but either works